ek9000App/src/devEK9000.h

```diff
@@ -83,24 +83,21 @@
 class devEK9000;
 class devEK9000Terminal;
 
-extern std::list<devEK9000*> g_Devices;
-extern bool g_bDebug;
-
 std::list<devEK9000*>& GlobalDeviceList();
 
 #define TERMINAL_FAMILY_ANALOG 0x1
 #define TERMINAL_FAMILY_DIGITAL 0x2
 
 #define DevInfo(fmt, ...)                                                                                              \
-	if (g_bDebug) {                                                                                                    \
+	if (devEK9000::debugEnabled) {                                                                                                    \
```

Comment:
`clang-format` is not happy here according to CI... https://github.com/slaclab/epics-ek9000/actions/runs/3057306552/jobs/4932323207#step:3:42